jfrog-cli/jfrog/artifactory_test.go

```diff
@@ -944,6 +944,27 @@ func TestExcludeBrokenSymlink(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+// Testing uploading broken symlinks.
+// Uploaded symlinks are not verified by default, so all symlinks should be uploaded
```

Comment:
I think that **by default** should be removed from the comment, because symlinks are never verified by default.